glue_jupyter/bqplot/profile/tests/test_viewer.py

```diff
@@ -157,3 +161,80 @@ def test_unit_conversion(app):
 
     assert len(d2.subsets) == 1
     assert_equal(d2.subsets[0].to_mask(), [0, 1, 1])
+
+
+def test_composite_xrange(app):
+    session = app.session
+
+    wcs2 = WCS(naxis=1)
+    wcs2.wcs.ctype = ['WAVE']
+    wcs2.wcs.crval = [10]
+    wcs2.wcs.cdelt = [10]
+    wcs2.wcs.crpix = [1]
+    wcs2.wcs.cunit = ['cm']
+
+    d2 = Data(f2=[2000, 1000, 3000])
+    d2.get_component('f2').units = 'mJy'
+    d2.coords = wcs2
+
+    data_collection = session.data_collection
+    data_collection.append(d2)
+
+    viewer = app.profile1d(data=d2)
+    tool = viewer.toolbar.tools['bqplot:xrange']
```

Comment:
We need to test the tool with brushing because `viewer.apply_roi(...)` will not trigger the bug.